src/bytes_ser_de.rs

```diff
@@ -108,19 +122,6 @@ impl<'a> Deserializer<'a> {
         Ok(buf)
     }
 
-    /// Reads all the remaining bytes.
-    #[inline]
-    pub fn value(&mut self) -> Result<Vec<u8>, CryptoCoreError> {
```

Comment:
shouldn't we move to 5.0.0 as we are removing public API?

Reply:
Yes we should :/

Reply:
We now use `Unreleased` in the changelog and don't change the version when pushing to `develop`. The version is fixed when pushing to `main`.